apps/ledger-live-desktop/tests/page/modal/send.modal.ts

```diff
@@ -42,27 +41,24 @@ export class SendModal extends Modal {
   @step("Enter recipient and tag")
   async fillRecipientInfo(transaction: Transaction) {
     await this.fillRecipient(transaction.accountToCredit.address);
-    if (transaction.memoTag) {
+    if (transaction.memoTag && transaction.memoTag !== "noTag") {
```

Comment:
first check in `if` seems sufficient, isn't it? or is `transaction.memoTag !== "noTag"` used for some specific condition?

Reply:
The condition is needed due to the new modal (on Stellar, Cosmos, Cardano) when you try to send funds without adding a tag.